app/models/question.rb

```diff
@@ -0,0 +1,20 @@
+# == Schema Information
+#
+# Table name: questions
+#
+#  id           :bigint           not null, primary key
+#  for_banks    :boolean
+#  for_partners :boolean
+#  title        :string
+#  created_at   :datetime         not null
+#  updated_at   :datetime         not null
+#
+class Question < ApplicationRecord
+  has_rich_text :content
```

Comment:
```suggestion
  has_rich_text :answer
```
I feel like `answer` is more applicable here since the activerecord is `Question`

Reply:
I renamed it to answer.